pkg/state/state.go

```diff
@@ -957,7 +1007,7 @@ func (st *HelmState) PrepareCharts(helm helmexec.Interface, dir string, concurre
 				skipDepsGlobal := opts.SkipDeps
 				skipDepsRelease := release.SkipDeps != nil && *release.SkipDeps
 				skipDepsDefault := release.SkipDeps == nil && st.HelmDefaults.SkipDeps
-				skipDeps := !isLocal || skipDepsGlobal || skipDepsRelease || skipDepsDefault
+				skipDeps := !isLocal || skipDepsGlobal || skipDepsRelease || skipDepsDefault || !isOCI
```

Comment:
Hi @chrismellard Why skip deps for non-OCI releases? I'm running into some issues with this change, I'm not familiar with OCI, can you explain your intention? Thank you!

Reply:
@nauxliu I think we should invert it, so that dep updates are skipped when the chart is from an OCI repo(i.e. it's a remote chart). The fix is included in #1667 and I'd appreciate if you could review it!